### ISSUE TITLE
Uncaught error when voting from the staking overview

### ISSUE THREAD

**Issue by Cyrelb** [chainx-org/chainx-dapp-wallet-bug#9]
1. Go to staking overiew
![1](https://user-images.githubusercontent.com/27161321/104508948-49460a80-55e9-11eb-8148-ee9711316bbb.jpg)
2. Click vote on the selected validator
![image](https://user-images.githubusercontent.com/27161321/104509064-6e3a7d80-55e9-11eb-92b1-943046765252.png)

**Comment by Deelux** [comment]
Yup, I have seen this too. It shows up on my side a few seconds after I have clicked "Vote" on the Staking Overiew page. The vote page does show up for a split second, but then it only shows the above error.
